ci-operator/step-registry/assisted/nutanix/setup/machine/assisted-nutanix-setup-machine-commands.sh

```diff
@@ -33,6 +33,18 @@ iso_name = "assisted-test-infra-machine-template"
 build_id = "${BUILD_ID}"
 EOF
 
+# Terraform Params
+# move to new ENV value with auto load
+export CI_CREDENTIALS_DIR=/var/run/vault/assisted-ci-vault
+
+for file in $CI_CREDENTIALS_DIR/TF_VAR_*; do
+   if [[  "$file" == *"TF_VAR_"* ]]; then
```

Comment:
why both `if` and `/TF_VAR_*` ?

Reply:
yes you right, This was for previous version  - i'll  fix it

Reply:
Thanks !